frontend/src/components/AuthUtils/index.test.js

```diff
@@ -0,0 +1,79 @@
+import authUtils from './index.js';
+import { getCurrentUser } from './AuthUtilsConsumer.js';
+import { SIGN_IN } from '../../constants/routes.js';
+
+const mockDisplayName = 'Sam Smith';
+const mockEmail = 'samsmith@email.com';
+const mockPhotoURL = 'http://www.photo-hosters.com/qwerty';
+const mockUid = '1234567890';
+const mockAuthenticatedUser = {
+  displayName: mockDisplayName,
+  email: mockEmail,
+  photoURL: mockPhotoURL,
+  uid: mockUid
+};
+const mockUnauthenticatedUser = null;
```

Comment:
This is different from what the real code could return. The real code doesn't have a default value set for the user -- should that be updated to be null like this test?

Reply:
I'm actually grabbing the value of the currentUser from [`Auth/AuthContext.js`](https://github.com/googleinterns/step53-2020/blob/auth-utils/frontend/src/components/Auth/AuthContext.js) using React Context. It already has currentUser set as null by default there.

Just to be safe though, I set `currentUser` in `AuthUtils/AuthUtilsConsumer.js` to have a default value of null as well.